static/app/views/dashboards/datasetConfig/spans.tsx

```diff
@@ -35,16 +32,18 @@ import {
 import {DisplayType, type Widget, type WidgetQuery} from 'sentry/views/dashboards/types';
 import {eventViewFromWidget} from 'sentry/views/dashboards/utils';
 import {EventsSearchBar} from 'sentry/views/dashboards/widgetBuilder/buildSteps/filterResultsStep/eventsSearchBar';
+import type {FieldValueOption} from 'sentry/views/discover/table/queryField';
+import {FieldValueKind} from 'sentry/views/discover/table/types';
 import {generateFieldOptions} from 'sentry/views/discover/utils';
 
 const DEFAULT_WIDGET_QUERY: WidgetQuery = {
   name: '',
-  fields: ['span.op', 'count(span.op)'],
+  fields: ['span.op', 'avg(span.duration)'],
```

Comment:
There isn't a finalized default yet, I'm just modifying this because `span.op` is no longer a valid argument to `count()` after these changes, according to the behaviour in explore.